### DISCUSSION TITLE
Is that normal guys ?

### DISCUSSION THREAD

**Discussion by ElBulbol** [RamonUnch/AltDrag#540] [General]
![Screenshot 2024-05-20 205142](https://github.com/RamonUnch/AltSnap/assets/142834487/8613dc21-c7e9-4a6f-a81b-1a08f525ab12)

**Comment by RamonUnch** [comment]
Pleass read description of the project at least.
https://github.com/RamonUnch/AltSnap/?tab=readme-ov-file#virustotal-false-positive

AltSnap makes a LowLevel hook of mouse and keyboard so it can look suspicious to some heuristics anti-viruses uses.
